package/network/services/picomdnsd/files/picomdnsd.init

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+USE_PROCD=1
+
+APPBINARY=/usr/sbin/picomdnsd
+LANIPADDR=$(/sbin/ifconfig "br-lan" | grep "inet " | awk -F' ' '{print substr($2,6)}')
```

Comment:
I tried using UCI but it can't determine IP when it's set to use DHCP, perhaps I'm missing something?

```
root@OpenWrt:~# uci get network.lan.ipaddr
uci: Entry not found
```

As you can tell, I'm not very good at init.d scripting at all ;-)